src/Raven.Studio/typescript/viewmodels/shell/about.ts

```diff
@@ -339,9 +344,41 @@ class about extends viewModelBase {
         shell.openFeedbackForm();
     }
 
+    private getConnectivityToLicenseServer() {
+        this.spinners.connectivityToLicenseServer(true);
+        return new getConnectivityToLicenseServerCommand()
+            .execute()
+            .done((connectionResult: Raven.Server.Web.Studio.LicenseHandler.ConnectivityToLicenseServer) =>
+                this.connectedToLicenseServer(connectionResult.StatusCode !== "OK"))
+            .always(() => this.spinners.connectivityToLicenseServer(false));
+    }
+
+    refreshConnectivityToLicenseServer() {
+        this.spinners.connectivityToLicenseServer(true);
+        const cmd = new getConnectivityToLicenseServerCommand();
+        cmd.execute()
+            .done((connectionResult: Raven.Server.Web.Studio.LicenseHandler.ConnectivityToLicenseServer) =>
+                this.connectedToLicenseServer(connectionResult.StatusCode === "OK"))
+            .always(() => this.spinners.connectivityToLicenseServer(false));
+    }
+
     activate(args: any) {
         super.activate(args, true);
-        return $.when<any>(this.getLicenseConfigurationSettings(), this.pullLatestVersionInfo(), license.fetchLicenseStatus());
+        return $.when<any>(this.getLicenseConfigurationSettings(),
+                           this.pullLatestVersionInfo(),
+                           license.fetchLicenseStatus()),
+                           this.getConnectivityToLicenseServer();
+    }
+    
+    compositionComplete() {
+        super.compositionComplete();
+        
+        popoverUtils.longWithHover($(".not-connected"),
+            {
+                content:
+                    `<small>Allow your server access to <code>api.ravendb.net</code><br>or contact support for an updated license.</small>`,
```

Comment:
there might be different causes then access - (slow network , another issues)

Reply:
=>
Unable to reach the RavenDB License Server at <code>api.ravendb.net</code>